service/service.go

```diff
@@ -22,6 +22,7 @@ type Feeder interface {
 
 type Broker interface {
 	Account() (model.Account, error)
+	PositionAllPairs(pairs []string) (assets []model.Assets, err error)
```

Comment:
I think we don't need this function. We can use the `Account()` which return all pairs balance. Using Account you optimize the result getting all pairs balance once.

Reply:
thanks for your reviewed, I will revise follow this one.